apps/system/fxa/test/marionette/fxa_launch_test.js

```diff
@@ -20,6 +20,9 @@ marionette('Firefox Accounts Launch Tests', function() {
           },
           settings: {
             'lockscreen.enabled': false
+          },
+          apps: {
+            'test-fxa-client.gaiamobile.org': + __dirname + '/test-fxa-client',
```

Comment:
I don't think this is necessary as linters are currently passing. The system .jshintrc file already defines the environment as node: https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/marionette/.jshintrc#L3

(This is something we should probably do everywhere)